packages/website/src/css/custom.css

```diff
@@ -8,7 +8,7 @@
 @import './prism.css';
 
 /* You can override the default Infima variables here. */
-html:root {
+html {
```

Comment:
Except, now something in either this change or the docusaurus bump is causing aXe to think the page has some element(s) with lower color contrast than needed though: https://github.com/typescript-eslint/typescript-eslint/runs/5581627126?check_suite_focus=true

![Screenshot of "Index -- has no accessibility issues detected by aXe (failed)" with accessibility violations on aria-command-name and color-contrast](https://user-images.githubusercontent.com/3335181/158806211-79a632e8-e6ee-4d50-8874-8a724097398e.png)

Reply:
Let me try reverting this first.